### PR TITLE
feat: Add no-data, remove borders from responsive mode

### PR DESCRIPTION
**Pull request by JKMarkowski**
## Related Issue
part of https://github.com/SAP/fundamental-ngx/issues/4622
part of https://github.tools.sap/dxp/jukebox/issues/420

## Description
There are
- added no-data mode
- responsvie mode borders removed
- word break css included into cells

#### Please check whether the PR fulfills the following requirements

1. The output matches the design specs
- [x] All values are in `rem`
- [x] Text elements follow the truncation rules
2. The code follows fundamental-styles code standards and style
- [x] only one top level `fd-*` class is used in the file
- [x] BEM naming convention is used
- [x] Mixins are used for repeatable code (`fd-rtl`, `fd-ellipsis`, `fd-flex`, `fd-selected`, `fd-focus`, ect.)
- [x] A11y support - keyboard support, screenreader support, proper ARIA attributes, etc.
- [x] `fd-reset()` mixin is applied to all elements
- [x] Variables are used, if some value is used more than twice.
- [x] Checked if current components can be reused, instead of having new code.
3. Testing
- [x] tested Storybook examples with "CSS Resources" `normalize` option 
- [x] tested Storybook examples with "CSS Resources" `unnormalize` option 
- [x] Verified all styles in IE11
- [x] Updated tests
- [x] last commit message should have `[ci visual]` so it can trigger chromatic visual regression (e.g. `test: run chromatic visual regression [ci visual]`)
4. Documentation
- [x] Storybook documentation has been created/updated
- [x] Breaking Changes [wiki](https://github.com/SAP/fundamental-styles/wiki/Breaking-Changes) has been updated in case of breaking changes.

### CODE REVIEW

**Review by itmilos — APPROVED** [review]

**Comment by InnaAtanasova** [comment]
> Screen Shot 2021-03-10 at 10 34 30 AM
> 
> Screen Shot 2021-03-10 at 10 34 13 AM
> 
> word-break: break-all will break the overflow rules. There might be cases where the user will want to add `...` and not break the words

this might be an option when max-width of the cell is provided

**Comment by JKMarkowski** [comment]
@InnaAtanasova changes that we agreed on are applied

File: src/table.scss [hunk 1]
```diff
@@ -124,6 +124,16 @@ $block: #{$fd-namespace}-table;
     }
   }
 
+  &__text {
+    word-break: break-word;
+    overflow: hidden;
+    text-overflow: ellipsis;
```
Comment (InnaAtanasova) [inline src/table.scss:130]: you could use the mixin here

**Review by InnaAtanasova — APPROVED** [review]

File: src/table.scss (hunk 1 above)
Comment (InnaAtanasova) [reply src/table.scss:130]: @itmilos why did you resolve this comment?

Comment (InnaAtanasova) [reply src/table.scss:130]: One of the Definitions of Done is `Mixins are used for repeatable code (fd-rtl, fd-ellipsis, fd-flex, fd-selected, fd-focus, ect.)` and there's a mixin `fd-ellipsis` that has these 3 rules

**Pull request merged**